apps/meeting/check-version.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs-extra');
```

Comment:
We do not need this anymore :)

Reply:
Do you mean to delete the whole `check-version.js` file?

Reply:
Yep,

Reply:
Yep, removed it